### PR TITLE
Add testmon to action

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

- Add testmon to action

### Changed

- Split coverage and unit tests actions

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
@seallard @ChrOertlin @diitaz93 @Vince-janv Seems like you have to sign up and pay for the actions to work (currently in beta testing). However, it does work locally. What do you think of signing up  or just doing it locally for now

**Comment by ChrOertlin** [comment]
Running the tests locally is quite speedy, the highest gain would be on github actions I think. Would it not be better to first streamline the tests and then check whether this is needed? 

Also it is quite often that I encounter untested functions. Would running less tests then not mask potential errors?

**Comment by diitaz93** [comment]
Yes, I also think that the increase in speed is not that urgent locally, the gain will be on GitHub. I would like to try it!

**Comment by Vince-janv** [comment]
> @seallard @ChrOertlin @diitaz93 @Vince-janv Seems like you have to sign up and pay for the actions to work (currently in beta testing). However, it does work locally. What do you think of signing up or just doing it locally for now

Where do you get the prompt/message to sign up?

**Comment by seallard** [comment]
This would be really nice 💯  I think we should try it out 👍

File: .testmondata
Comment (seallard) [inline .testmondata:1]: We dont want to check in these binary files - lets update the .gitignore to not track them

File: .testmondata-shm
Comment (seallard) [inline .testmondata-shm:1]: We dont want to check in these binary files - lets update the .gitignore to not track them

File: .testmondata-wal
Comment (seallard) [inline .testmondata-wal:1]: We dont want to check in these binary files - lets update the .gitignore to not track them

**Review by seallard — APPROVED** [review]
Nice, we just want to update .gitignore with the binary files testmon generates:

- .testmondata-wal
- .testmondata-shm
- .testmondata

**Comment by henrikstranneheim** [comment]
No immediate need for this and there is a cost involved.

Refactoring the tests and updating packages as well as switching to python 3.11 (10-60%) faster should be done first.

**Pull request closed without merging**